src/99-appendices/11-arterial-spin-labeling.md

```diff
@@ -0,0 +1,105 @@
+
+### Case 1: `*_asl.nii[.gz]` consists of volume_types `control`, `label`
+
+In most cases, the ASL timeseries, provided by the scanner, consist of a series of `control` and `label`, and optionally `m0scan`, volumes. 
+In this case, only the `control`, `label`, and optionally `m0scan` volumes should be stored in the `*_asl.nii[.gz]`, 
+and the exact volume_type series should be specified in the `*_aslcontext.tsv`. Example of `*_aslcontext.tsv`:
+
+```Text
+volume_type
+control
+label
+control
+label
+m0scan
+```
+
+### Case 2: `*_asl.nii[.gz]` consists of volume_types `deltam` (scanner does not export `control` or `label` volumes)
+
+In some cases, `control` and `label` volumes are lacking within the acquired ASL timeseries,
+but the intermediate `deltam` - and optionally an `m0scan` -
+volume is reconstructed/exported by the scanner.
+In this case, the `deltam` should be included in the `*_asl.nii[.gz]` and specified in the `*_aslcontext.tsv`.
+Example of `*_aslcontext.tsv`:
+
+```Text
+volume_type
+deltam
+m0scan
+```
+
+### Case 3: `*_asl.nii[.gz]` consists of volume_type `cbf` (scanner does not export `control`, `label`, or `deltaM` volumes)
+
+If `control` and `label` or intermediate ASL volumes are not reconstructed or exported,
+but a pre-calculated `cbf` - and optionally a `m0scan` - volume is provided by the scanner,
+the `cbf` should be included in the `*_asl.nii[.gz]` and specified in the `*_aslcontext.tsv`.
+Example of `*_aslcontext.tsv`:
+
+```Text
+volume_type
+cbf
+m0scan
+```
+
+## Summary Image of the most common ASL sequences
+
+The following images illustrate the main BIDS metadata fields for three `ArterialSpinLabelingType`'s: `CASL`, `PCASL`, and `PASL`. Images are courtesy of, and adapted with
+permission from Y. Suzuki and [OSIPI Task force 4.1: ASL lexicon milestone 1](https://www.osipi.org/task-force-4-1/)".
+
+### (P)CASL sequence
+
+![PCASL](../04-modality-specific-files/images/asl_pcasl_sequence.png)
+
+For (P)CASL, specifying the `LabelingDuration` and the `PostLabelingDelay` is required. The `LabelingDuration` is defined as the total duration of the labeling pulse train, in
+seconds. `PostLabelingDelay` is the time, in seconds, after the end of the labeling until the middle of the excitation pulse applied to the imaging slab (for 3D acquisition) or
+first slice (for 2D acquisition). Additionally, the `BackgroundSuppressionPulseTime`'s is required in case `BackgroundSuppression` was applied. This an array of numbers
+containing the timing, in seconds, of the background suppression pulses with respect to the start of the labeling. In the case of `PCASL`, the recommended `PCASLType` field
+defines the type of the gradient pulses used in the `control` condition (`balanced` or `unbalanced`), whereas, in case of `CASL`, the recommended `CASLType` field describes if a
+separate coil is used for labeling (`single-coil` or `double-coil`).
```

Comment:
```suggestion
For (P)CASL, specifying the `LabelingDuration` and the `PostLabelingDelay` is required.
The `LabelingDuration` is defined as the total duration of the labeling pulse train in seconds.
`PostLabelingDelay` is the time in seconds after the end of the labeling until the middle of the excitation pulse applied
to the imaging slab (for 3D acquisition) or first slice (for 2D acquisition).
Additionally, the `BackgroundSuppressionPulseTime`'s is required in case `BackgroundSuppression` was applied.
This an array of numbers containing the timing in seconds of the background suppression pulses
with respect to the start of the labeling.
In the case of `PCASL`, the recommended `PCASLType` field defines the type of the gradient pulses
used in the `control` condition (`balanced` or `unbalanced`),
whereas in case of `CASL`,the recommended `CASLType` field describes if a separate coil is used for labeling
(`single-coil` or `double-coil`).
```